<?xml version="1.0"?>
<launch>
  <arg name="gps_dev" default="/dev/ttyUSB1"/>
  <arg name="imu_dev" default="/dev/ttyUSB0"/>
  <arg name="ardino_dev" default="/dev/ttyACM0"/>
	<include file="$(find truck_base)/launch/static_tf.launch" />
  <node pkg="truck_base" type="truck_base_node" name="truck_base_node" output="screen"/>
  <node pkg="truck_base" type="twist_sim.py" name="twist_sim" />
	<include file="$(find truck_base)/launch/localization_odom.launch" />
	<include file="$(find truck_base)/launch/imu.launch">
    <arg name="imu_dev" value="$(arg imu_dev)"/>
  </include>
  <include file="$(find truck_base)/launch/gps.launch">
    <arg name="gps_dev" value="$(arg gps_dev)"/>
  </include>
	<include file="$(find truck_base)/launch/arduino.launch" />
</launch>
